installation-and-deployment/deploying-with-helm.md

```diff
@@ -0,0 +1,59 @@
+# Deploying on Kubernetes with Helm Chart
+
+To deploy Aqueduct on a Kubernetes cluster, you can use the helm chart we've created. Please follow the steps below.
+
+### Prerequisites
+
+* Follow the instruction [here](https://helm.sh/docs/intro/install/) to install Helm on your local machine.
+* You need to have access to an existing Kubernetes cluster. This means your local machine needs to have a kubeconfig file (typically located in `~/.kube/config`) that points to your cluster.
+* Make sure [`kubectl`](https://kubernetes.io/docs/tasks/tools/) is installed on your local machine.
+
+### Setting Up
+
+1. Add the repository to your local Helm manager.
+   ```
+   helm repo add aqueduct https://aqueducthq.github.io/aqueduct-helm/
+   ```
+
+2. Sync versions available in the repo to your local Helm cache.
+   ```
+   helm repo update
+   ```
+
+3. Search for the latest stable versions of the charts found.
+   ```
+   helm search repo aqueduct
+   ```
+
+4. Install the Helm chart.
+   ```
+   helm install aqueduct/aqueduct --generate-name
+   ```
+
+### Customizing Install Options
+
+The installation command above deploys the latest version of Aqueduct in the default namespace with the server running with Python 3.8. The name of the deployment is generated with the `--generate-name` flag. You can customize all of these by running the following:
```

Comment:
This is a little confusing right now. Let's say that by default, we select Python 3.8.